Bigtable/tests/System/BackupTests.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Tests\System;
+
+use Google\Cloud\Bigtable\Admin\V2\Backup;
+use Google\Protobuf\Timestamp;
+use Google\ApiCore\ApiException;
+
+/**
+ * @group bigtable
+ * @group bigatableBackup
+ */
+class BackupTests extends BigtableTestCase
+{
+    private static $destinationClusterId;
+    private static $destinationProjectId;
+    private static $backupId;
+    private static $backupName;
+    private static $copyBackupId;
+    private static $expireTime;
+
+    public const DESTINATION_LOCATION_ID = 'us-central1-b';
+
+    public static function set_up_before_class()
+    {
+        parent::set_up_before_class();
+        self::$destinationClusterId = uniqid(self::CLUSTER_ID_PREFIX);
+        $keyFilePath = getenv('GOOGLE_CLOUD_PHP_WHITELIST_TESTS_KEY_PATH');
+        $keyFileData = json_decode(file_get_contents($keyFilePath), true);
+        self::$destinationProjectId = $keyFileData['project_id'];
+        self::$backupId = uniqid('backup');
+        self::$copyBackupId = 'copy' . self::$backupId;
+        self::$expireTime = new Timestamp();
+        self::$expireTime->setSeconds(time() + 86400);
+        self::$backupName = self::$tableAdminClient->backupName(
+            self::$projectId,
+            self::$instanceId,
+            self::$clusterId,
+            self::$backupId
+        );
+    }
+
+    public function testCreateBackup()
+    {
+        $backup = new Backup();
+
+        $sourceTableName = self::$tableAdminClient->tableName(
+            self::$projectId,
+            self::$instanceId,
+            self::TABLE_ID
+        );
+        $backup->setSourceTable($sourceTableName);
+        $backup->setExpireTime(self::$expireTime);
+
+        $parentName = self::$instanceAdminClient->clusterName(
+            self::$projectId,
+            self::$instanceId,
+            self::$clusterId
+        );
+        $operationResponse = self::$tableAdminClient->createBackup(
+            $parentName,
+            self::$backupId,
+            $backup
+        );
+        $operationResponse->pollUntilComplete();
+        $result = $operationResponse->getResult();
+        self::assertStringContainsString(self::$backupName, $result->getName());
+        self::assertEquals("", $result->getSourceBackup());
```

Comment:
There's no `getSourceBackup(...)` in `Google\Cloud\Bigtable\Admin\V2\Backup`. Did you intend to write like this instead?

```suggestion
        self::assertStringContainsString(self::TABLE_ID, $result->getSourceTable());
```

Reply:
This getter will be ammended to the backup class from the new proto changes. 

I've added the proto changes to be introduces after this feature above in the [description](https://github.com/googleapis/google-cloud-php/pull/5651#:~:text=in%20preview%20branch.-,Proto,-changes%20/%20additions)